packages/config/config/devices/0x0063/55258_zw4002.json

```diff
@@ -31,9 +31,34 @@
 		}
 	},
 	"paramInformation": [
+		{
+			"#": "3",
+			"$import": "~/templates/master_template.json#led_indicator_four_options"
```

Comment:
@jascoproducts you used the 4-option template here which does not match your documentation in https://github.com/jascoproducts/firmware/issues/50#issuecomment-3406906633.  I can fix this in #8359 if you can confirm the 3-option inverted template is actually the correct one.